recipes/default.rb

```diff
@@ -17,44 +17,48 @@
 # limitations under the License.
 #
 
+include_recipe 'yum-epel' if platform_family?('rhel')
+
 package "ucarp"
 
-template "/etc/network/interfaces" do
-  source "interfaces.erb"
-  owner "root"
-  group "root"
-  mode 0644
-  variables(
-    :vid =>  node['ucarp']['vid'],
-    :vip =>  node['ucarp']['vip'],
-    :netmask => node['ucarp']['netmask'],
-    :password => node['ucarp']['password'],
-    :advskew => node['ucarp']['master'] ? node['ucarp']['advskew'] : node['ucarp']['advskew'] + 99,
-    :advbase => node['ucarp']['advbase'],
-    :master => node['ucarp']['master'] ? 'yes' : 'no',
-    :interface => node['ucarp']['interface'],
-    :bonded_interfaces => node['ucarp']['bonded_interfaces'],
-    :bond_mode => node['ucarp']['bond_mode']
-  )
-  notifies :restart, "service[networking]", :immediately
+unless platform_family?('rhel')
+  template "/etc/network/interfaces" do
```

Comment:
Thats from the original cookbook. I'm only concerned right now about centos. We can fix it later.